css/style.css

```diff
@@ -0,0 +1,293 @@
+body {
+    color: #000000;
+    font-family: 'montserrat-normal', sans-serif;
+}
+
+.title,
+.bag__title-bold,
+.bag__title,
+.bag-title-bold,
+.bag-title,
+.name {
+    font-family: 'montserrat-bold', sans-serif;
+}
+
+/* 1 section*/
+
+.new-collection__background {
+    position: absolute;
+    max-width: 461px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Love-metal_LOCO/pullRequest?prid=5388782)